app/smc/prj.conf

```diff
@@ -58,3 +58,7 @@ CONFIG_RETENTION_BOOTLOADER_INFO_TYPE_MCUBOOT=y
 # Generate unsigned images by default
 # To sign FW, use SB_CONFIG_BOOT_SIGNATURE_KEY_FILE="/path/to/key.pem"
 CONFIG_MCUBOOT_GENERATE_UNSIGNED_IMAGE=y
+
+# DMA
+CONFIG_DMA=y
```

Comment:
I think we already have this defined at the board level (just disabled for mcuboot), so it can be removed.